packages/@aws-cdk/aws-eks/test/integ.eks-cluster.ts

```diff
@@ -49,8 +48,6 @@ class EksClusterStack extends TestStack {
 
     this.assertSpotCapacity();
 
-    this.assertInferenceInstances();
```

Comment:
I moved this assertion to its own separate test because its a little flaky, and makes running our canonical test harder.